assignments/redisish.md

```diff
@@ -27,6 +16,21 @@ A missing newline is an error, everything after the first newline can be ignored
 
 Empty messages are allowed.
 
-## Hints
 
-Read the docs for `str` (primitive), especially `splitn` and `trim`.
+
+1. Create a library project called `redisish`.
+2. Read the docs for `str` (primitive), especially `splitn` and `trim`. Pay attention to their return type.
+
+3. Implement the following function, so it parses the input according to the rules of the protocol, described above.
+
+```rust
+// appropriate data structures
+
+pub fn parse(input: &str) -> Result<Command, Error> {
+    ///...
+}
+
+```
+Use `enums` for the data structures. One for the Commands and one for the different error types. Use `if let` and `match` for control flow.
+
+4. Test as appropriate. (What is appropriate?)
```

Comment:
Is that a mental note?

Reply:
Ah yes. Either appropriate explains itself, and then the "Test program" would be sufficient, or it needs explaining, what appropriate is.

Reply:
It should be suite of tests that sufficiently checks whether the rules above have been implemented.